src/ATen/CMakeLists.txt

```diff
@@ -63,30 +65,31 @@ IF ($ENV{TH_BINARY_BUILD})
   SET(CMAKE_CXX_FLAGS "-static-libstdc++ ${CMAKE_CXX_FLAGS}")
 ENDIF()
 
-IF(NO_CUDA)
-  MESSAGE(STATUS "ignoring CUDA")
+IF(NOT NO_CUDA)
+  FIND_PACKAGE(CUDA 5.5)
+ENDIF()
+
+IF(NO_CUDA OR NOT CUDA_FOUND)
+	MESSAGE(STATUS "ignoring CUDA")
```

Comment:
I am not a fan of this. If they don't specify `NO_CUDA=1` and CUDA cannot be found, this should be a compilation error, not a warning.